### ISSUE TITLE
Log parsing functionality

### ISSUE THREAD

**Issue by GeckoEidechse** [R2NorthstarTools/FlightCore#157]
Parse logs to extract certain useful information like

- [ ] Get installed mods
- [ ] Get enabled mods
- [ ] Get Northstar version number
- [ ] check for errors/stacktraces
- [ ] ???

**Comment by taskinoz** [comment]
I made a tool to parse NorthStar logs that have a bunch of those already, so you might be able to use some of the regex patterns in this.

https://github.com/taskinoz/Northstar-Logging-Utility/blob/main/src/App.js

**Comment by GeckoEidechse** [comment]
Good point, forgot that you already put some work into this. Thanks for the heads-up <3
